src/simtools/production_configuration/limits_calculation.py

```diff
@@ -0,0 +1,161 @@
+"""Calculate the thresholds for energy, radial distance, and viewcone."""
+
+import numpy as np
+
+
+class LimitCalculator:
+    """
+    Class to compute thresholds/limits for energy, radial distance, and viewcone.
```

Comment:
"Compute thresholds/limits..."

(clear that it is a class, see two lines higher...)

Reply:
Maybe also add to the help here that what kind of histograms are read and they have been generated with sim_telarray.